indra/llrender/llshadermgr.cpp

```diff
@@ -79,7 +79,7 @@ bool LLShaderMgr::attachShaderFeatures(LLGLSLShader * shader)
     //////////////////////////////////////
 
     // NOTE order of shader object attaching is VERY IMPORTANT!!!
-    if (features->calculatesAtmospherics)
+    if (features->calculatesAtmospherics || features->hasGamma || features->isDeferred)
```

Comment:
shouldn't isDeferred always be on now?

Reply:
It's a separate shader feature flag controlling deferredUtil shader helpers being linked or not.